extractor/docker.go

```diff
@@ -127,6 +128,19 @@ func (d DockerExtractor) createRegistryClient(ctx context.Context, domain string
 	})
 }
 
+func (d DockerExtractor) SaveLocalImage(ctx context.Context, imageName string) (io.ReadCloser, error) {
+	cli, err := client.NewClientWithOpts(client.FromEnv)
+	if err != nil {
+		return nil, xerrors.Errorf("error in docker NewClient")
```

Comment:
It seems good enough to use xerrors.New.

Reply:
Thanks!